prow/plugins/lgtm/lgtm.go

```diff
@@ -341,6 +341,17 @@ func handlePullRequest(log *logrus.Entry, gc githubClient, config *plugins.Confi
 
 	// rc.repo.Owner.Login, rc.repo.Name)
 	opts := optionsForRepo(config, org, repo)
+	if opts.StickyForCollaborators {
+		if ok, merr := gc.IsCollaborator(org, repo, pe.PullRequest.User.Login); merr == nil && ok {
+			// StickyForCollaborators indicates if LGTM is sticky for PRs authored by collaborators.
```

Comment:
This seems like the completely wrong place to document why this feature exists. IMHO, code comments should document what the code is trying to accomplish and the config / actual documentation (markdown) comments should document why the feature exists...

Reply:
There is documentation on the config struct too, this is just redundant. I agree that it could be removed, but I'm also fine with it staying to clarify the risk related to this option.

This might be a feature to avoid documenting in markdown given that we want to strongly discourage its use?
/shrug